### ISSUE TITLE
[Prerelease] Can't update repositories

### ISSUE THREAD

**Issue by Olympic1** [KSP-CKAN/CKAN#2387]
Background
----------

**CKAN Version:** 
v1.25.0-PRE1

**KSP Version:** 
1.4.1.2089

**Operating System:** 
Windows 10

**Have you made any manual changes to your GameData folder (i.e., not via CKAN)?** 
No

Problem
-------

**What steps did you take in CKAN?**
* Launched CKAN
* Clicked refresh button

**What did you expect to happen?**
That it updates the repositories

**What happened instead?**
Connection couldn't be established and progress tab stays open forever untill you press `Cancel`
Tested with the latest release (v1.24.0) and there it works fine.

**CKAN error codes (if applicable):** 
```
Updating repositories...
Connection to https://github.com/KSP-CKAN/CKAN-meta/archive/master.tar.gz could not be established.
Repository update failed!
```

**Comment by HebaruSan** [comment]
Are you possibly running CKAN-GUI.exe directly or via a debugger rather than the main ckan.exe? The fix for #2293 relies on code that sets a global variable in the runtime, which we do in `CKAN.CmdLine.MainClass.Main()` because it's the first thing that runs in the real EXE.

**Comment by Olympic1** [comment]
No, I ran the main ckan.exe. Rebuild the ckan.exe again and now the errors don't show up. Probably had some stale file(s).
